test/cjs-mocha-compiler.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var fs = require('fs');
+var traceurAPI = require('../src/node/api.js');
+
+var org = require.extensions['.js'];
+require.extensions['.js'] = function(module, path) {
+  if (!/\/node_modules\//.test(path)) {
+    var content = fs.readFileSync(path, 'utf8');
+    if (shouldSkip(path, content)) {
+      return;
+    }
+
+    // TODO(arv): Reuse parseProlog.
```

Comment:
The comment seems out of place here.  I guess you mean that the shouldSkip could use parseProlog?